03_simon_says/simon_says.rb

```diff
@@ -1 +1,43 @@
 #write your code here
+def echo(word)
+    "#{word}"
+end
+
+def shout(word)
+    "#{word.upcase}"
```

Comment:
Since `#upcase` is a string-specific method anyway, there isn't really benefit from using string interpolation here.

```suggestion
    word.upcase
```